app/src/main/java/com/checkin/app/checkin/utility/Utils.java

```diff
@@ -332,6 +331,23 @@ public static String formatDueTime(@NonNull Date startTime, @NonNull Date endTim
         return "Under 1 min";
     }
 
+    public static String formatOpenTimings(@NonNull String openTime, @NonNull String closeTime) {
```

Comment:
Hmm... can't see why it needs to be a utility function. I can't imagine this being reused.